frontend/src/App.js

```diff
@@ -52,9 +51,11 @@ function App() {
               <Route exact path="/courses/list" element={<CoursesIndexPage />} />
               <Route exact path="/courses/create" element={<CoursesCreatePage />} />
               <Route exact path="/personalschedules/details/:id" element={<PersonalSchedulesDetailsPage />} />
+
             </>
           )
         }
+        <Route exact path="/coursebyinstructor/search" element={<CourseByInstructorIndexPage />} />
         <Route exact path="/coursedescriptions/search" element={<CourseDescriptionIndexPage />} />
         <Route exact path="/courseovertime/search" element={<CourseOverTimeIndexPage />} />
         <Route exact path="/instructor/search" element={<CourseByInstructorIndexPage/>} />
```

Comment:
This line should be removed

Reply:
i think the /instructor/search one should be removed.